tests/testthat/test-bm-dataset-taxi-2013.R

```diff
@@ -0,0 +1,10 @@
+# Can't run benchmark because the data takes a long time to download (~1h), plus running is slow (~6m)
```

Comment:
We don't _have to_ this PR, but we could make a small sample of this dataset that we [include in the package](https://github.com/ursacomputing/arrowbench/blob/018ce79135460b9f8b64e341f877f8a0e1f6433f/R/known-sources.R#L61-L79)

Reply:
Oh, I didn't realize those were part of the package! Let me see if I can get it to play nice with datasets as opposed to files without a lot of work, and if I can I'll add it.

Reply:
K, implemented `test_datasets` parallel to `test_sources` and added `taxi_2013_sample`, which is still 12 files but only 1000 rows

Reply:
One request - can you include the code that you used to create the subset in a `data-raw` folder and then ignore that in the build?

Reply:
Pushed!